test/parallel/test-cli-nodeopt.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const common = require('../common');
+if (process.config.variables.node_without_node_options)
+  return common.skip('missing NODE_OPTIONS support');
+
+// Test options specified by env variable.
+
+const assert = require('assert');
+const exec = require('child_process').execFile;
+
+disallow('--version');
+disallow('-v');
+disallow('--help');
+disallow('-h');
+disallow('--eval');
+disallow('-e');
+disallow('--print');
+disallow('-p');
+disallow('-pe');
+disallow('--check');
+disallow('-c');
+disallow('--interactive');
+disallow('-i');
+disallow('--v8-options');
+disallow('--');
+
+function disallow(opt) {
+  exec(process.execPath, {env: {NODEOPT: opt}}, common.mustCall(function(err) {
+    const message = err.message.split('\n')[1];
+    const expect = process.execPath + ': ' + opt +
+      ' is not allowed in NODE_OPTIONS';
```

Comment:
Can you line this up with `process` on the previous line or make the indentation 4 spaces.